Rider may hang on loading screen due to exception

Recently I've upgraded Rider to 2020.3.4 with all plugins (including Azure Toolkit plugin). Rider hangs on loading screen with the following logs:
```
2021-03-22 08:49:28,813 | ERROR | PluginManager                  | JetBrains Rider 2020.3.4  Build #RD-203.7717.10
2021-03-22 08:49:28,814 | ERROR | PluginManager                  | JDK: 11.0.10; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2021-03-22 08:49:28,814 | ERROR | PluginManager                  | OS: Windows 10
2021-03-22 08:49:28,814 | ERROR | PluginManager                  | Plugin to blame: Azure Toolkit for Rider version: 3.46.0.1054-2020.3
2021-03-22 08:49:28,815 | ERROR | PluginManager                  | Last Action: 
2021-03-22 08:49:28,921 | INFO  | StubIndexImpl                  | Following stub indices will be built:dom.namespaceKey(v = 1),dom.elementClass(v = 0),properties.index(v = 2),Css.ampersand.selector(v = 2),Css.class(v = 2),Css.id(v = 2),Css.attr(v = 2),Css.custom.property(v = 2),Css.custom.mixin(v = 2),Less.mixins(v = 2),Less.variables(v = 2),Scss.variable(v = 2),Scss.placeholder.selector(v = 2),Scss.function(v = 2),Scss.mixin(v = 2),Stylus.variables(v = 2),Stylus.mixins(v = 2),markdown.header(v = 2),js.qualified.shortName(v = 4),js.element.qualifiedName(v = 3),JS.global.qualified.elements(v = 3),ts.external.module.index(v = 3),JS.class.super(v = 5),JS.class.implements(v = 3),JS.custom.stub.index(v = 2),js.symbol.index2(v = 6),js.typedef.index2(v = 2),js.generics.index(v = 2),JS.base.types.index(v = 3),js.class.index2(v = 3),js.namespace.index(v = 2),js.local.namespace.members.index(v = 2),JS.subclass.stub.index(v = 2),js.symbol.qualified.names.index(v = 1),amd.baseUrl.index(v = 2),amd.requirePaths.index(v = 2),js.global.symbol.index(v = 2),js.exported.index(v = 2),es6.exported.index(v = 2),js.imported.bindings.index(v = 2),es6.assignment.index(v = 2),ts.local.classes(v = 2),ts.embedded.content.index(v = 2),js.nonglobal.symbol.index(v = 2),js.lazy.packages(v = 2),js.gcl.modules(v = 2),js.doc.modules(v = 2),js.amd.modules(v = 2),jade.mixin(v = 2),vue.components.index(v = 25),vue.options.index(v = 25),vue.mixin.binding.index(v = 25),vue.extends.binding.index(v = 25),vue.global.directives.index(v = 25),vue.url.index(v = 25),vue.id.index(v = 25),vue.global.filters.index(v = 25),vuex.store.index(v = 25),angularjs.directives.index(v = 65),angularjs.directives.doc.index(v = 65),angularjs.controller.index(v = 65),angularjs.module.index(v = 65),angularjs.filter.index(v = 65),angularjs.symbol.index(v = 65),angularjs.ui.router.states.index(v = 65),angularjs.ui.router.generic.states.index(v = 65),angularjs.templateUrl.index(v = 65),angularjs.generic.modules.index(v = 65),angularjs.delimiter.index(v = 65),angular2.templateUrl.index(v = 9),angular2.source.pipe.index(v = 9),angular2.source.directive.index(v = 9),angular2.source.module.index(v = 9),angular2.ivy.pipe.index(v = 9),angular2.ivy.directive.index(v = 9),angular2.ivy.module.index(v = 9),angular2.metadata.pipe.index(v = 9),angular2.metadata.className.index(v = 9),angular2.metadata.directive.index(v = 9),angular2.metadata.node.index(v = 9),angular2.metadata.function.index(v = 9),angular2.metadata.module.index(v = 9),sql.table.shortName(v = 3),sql.column.name(v = 3),sql.names(v = 21)
2021-03-22 08:49:28,922 | INFO  | StubIndexImpl$StubIndexInitialization | Initialization done: 384
2021-03-22 08:49:29,334 | ERROR | ActionManagerImpl              |  [Plugin: com.intellij.resharper.azure]
com.intellij.diagnostic.PluginException:  [Plugin: com.intellij.resharper.azure]
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.instantiate(ActionManagerImpl.java:251)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.convertStub(ActionManagerImpl.java:221)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.getActionImpl(ActionManagerImpl.java:570)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.preloadActions(ActionManagerImpl.java:1621)
	at com.intellij.openapi.actionSystem.impl.ActionPreloader.preload(ActionPreloader.java:15)
	at com.intellij.openapi.application.Preloader.lambda$preload$0(Preloader.java:84)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:178)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:658)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:610)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:65)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:165)
	at com.intellij.openapi.application.Preloader.lambda$preload$1(Preloader.java:74)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:216)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:27)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:195)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:208)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:184)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.instantiate(ActionManagerImpl.java:241)
	... 23 more
Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.microsoft.azuretools.authmanage.AuthMethodManager$LazyHolder
	at com.microsoft.azuretools.authmanage.AuthMethodManager.getInstance(AuthMethodManager.java:80)
	at com.microsoft.azuretools.ijidea.actions.AzureSignInAction.<init>(AzureSignInAction.java:58)
	... 28 more
```

But Rider stopped behaving incorrectly after Azure Toolkit plugin was removed. Could you investigate this issue?
I also tried to re-install Rider, but it does not affect at all.

--- a/PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/platformPlugin.xml
+++ b/PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/platformPlugin.xml
@@ -6,7 +6,7 @@
     <html>
       <h4>Bug fix:</h4>
       <ul>
-        <li>Fix Rider may hang on loading screen due to exception (<a href="https://github.com/JetBrains/azure-tools-for-intellij/issues/459">#459</a>)</li>
+        <li>Don't output error message when not needed (<a href="https://github.com/JetBrains/azure-tools-for-intellij/pull/461">#461</a>)</li>
       </ul>
     </html>
     ]]>

--- a/PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/platformPlugin.xml
+++ b/PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/platformPlugin.xml
@@ -6,7 +6,7 @@
     <html>
       <h4>Bug fix:</h4>
       <ul>
-        <li>Fix Azure Toolkit for Rider: Deployment Slot setting is not saved (<a href="https://github.com/JetBrains/azure-tools-for-intellij/issues/423">#423</a>)</li>
+        <li>Fix Rider may hang on loading screen due to exception (<a href="https://github.com/JetBrains/azure-tools-for-intellij/issues/459">#459</a>)</li>
       </ul>
     </html>
     ]]>

--- a/Utils/azuretools-core/src/com/microsoft/azuretools/utils/CommandUtils.java
+++ b/Utils/azuretools-core/src/com/microsoft/azuretools/utils/CommandUtils.java
@@ -120,7 +120,7 @@ public class CommandUtils {
         executor.setExitValues(null);
         try {
             executor.execute(commandLine);
-            if (!mergeErrorStream) {
+            if (!mergeErrorStream && err.size() > 0) {
                 logger.log(Level.SEVERE, err.toString());
             }
             return out.toString();

